backend/project.clj

```diff
@@ -34,7 +34,7 @@
                  [com.google.gdata/core "1.47.1"]
                  [org.xerial/sqlite-jdbc "3.7.2"]
                  ; Akvo FLOW dependencies
-                 [org.akvo.flow/akvo-flow "v1.9.49-211-g94e20820" :classifier "classes"]
+                 [org.akvo.flow/akvo-flow "v1.9.49-193-g855fa8e4" :classifier "classes"]
```

Comment:
Are you sure about this? It doesn't seem right to change to an older version.

Reply:
Let's wait for @stellanl to comment on this ... My understanding is that this revision is 193 commits _ahead_ of the tag `v1.9.49`. That's why I asked about the relationship between `master` and `develop` in akvo-flow repo.

Reply:
When I made the edit 1.10 was not yet tagged.